### ISSUE TITLE
Change support confirmation to a modal

### ISSUE THREAD

**Issue by juliastarkov** [mozilla/fxa#2280]
We need a modal confirmation notification after a user successfully submits a support form. The key here is that we'll be able to pull the copy from the two dropdown form fields and include it in the message here. 

NOTE: We are still waiting on an illustration and copy approval. I will update this as soon as we get those things. 

![Support Form (Filled out)](https://user-images.githubusercontent.com/26334275/63389297-1ea64280-c360-11e9-861d-e1a6140ca94e.png)

**Comment by chenba** [comment]
Note that since we are displaying the modal on the same page as teh form, which is on the content server, we need to remove some code from the payments server for the current alert bar message.

@juliastarkov The alert bar is used for the payment info update messages.  I want to confirm that we are not changing those.

**Comment by juliastarkov** [comment]
@chenba Yes, we are only changing the notification UI for a support form confirmation.

**Comment by juliastarkov** [comment]
@chenba Here is the updated copy for the support modal, with two different scenarios for the MVP:

**Secure Proxy**
Header: Your message was sent to support
Body: Thank you for reaching out to Mozilla Support about <insert dropdown 2 selection> for <insert dropdown 1 selection>. We'll contact you via email as soon as possible.

**Other**
Header: Your message was sent to support
Body: Thank you for reaching out to Mozilla Support about <insert dropdown 2 selection>. We'll contact you via email as soon as possible.

NOTE 1: please make the dropdown selection copy in the modal lower case (not sentence case the way it appears in the dropdowns). 

NOTE 2: Zeplin and Invision are also up to date with the copy. 

(The animation is still being worked on and should be ready mid-September).

**Comment by chenba** [comment]
@juliastarkov can you include the link to Zeplin?

**Comment by chenba** [comment]
While the image in the issue description show an 'X' to close the modal, and presumably, clicking outside of the model also close the modal, we cannot do so as there are no further actions the user can take on the page.

Maybe we should remove that 'X', and do nothing when the user click outside the modal.

**Comment by chenba** [comment]
@juliastarkov sorry I didn't confirm with you before: what's the image above the modal heading?  I assumed it's the user's the profile avatar, but I realized I never really learned what it's supposed to be.

**Comment by chenba** [comment]
@juliastarkov Here's what the modal look like on desktop and mobile.  I'll refrain from adding the image above the heading until I get a confirmation on what it is.

![image](https://user-images.githubusercontent.com/198055/64045445-ed333100-cb2e-11e9-9d22-4a029198e498.png)
![image](https://user-images.githubusercontent.com/198055/64045983-418ae080-cb30-11e9-8672-7a9446f31127.png)

**Comment by chenba** [comment]
(No idea why the screen capture for the mobile image was scaled up.)

**Comment by juliastarkov** [comment]
@chenba 

- Link to zeplin: https://zpl.io/bAnmJQn

- Yes, good call on removing the X

- No, the image isn't the user avatar, it will be an animation to add delight to the modal (probably like a little thumbs up, or something)

- The screenshots look good, a couple of notes: (1) can we make the category input, in this case "Payment & billing" be in lower case like this, "payment & billing"? It looks kind of awkward in sentence case. (2) Please make the line spacing a little wider; the body copy looks very tight now.

**Comment by chenba** [comment]
>     * No, the image isn't the user avatar, it will be an animation to add delight to the modal (probably like a little thumbs up, or something)

Should it be file as a follow-up, or should it be part of this issue?  Either is fine; I just don't know the target timeline for this change or if we can have the modal without the image at first.


>     * The screenshots look good, a couple of notes: (1) can we make the category input, in this case "Payment & billing" be in lower case like this, "payment & billing"?

Yes.  Well, I can only be certain about English; I'll rely on translators to provide translated strings.

**Comment by chenba** [comment]
@juliastarkov here are the line height and sentence case changes:

![image](https://user-images.githubusercontent.com/198055/64266499-4a95fc00-cefa-11e9-89b9-836bf35b2476.png)

![image](https://user-images.githubusercontent.com/198055/64266399-2803e300-cefa-11e9-8be7-0411a2cf4604.png)

**Comment by juliastarkov** [comment]
@chenba These look great, thanks for implementing the changes. 

Regarding the illustration (sorry, accidentally called it "animation" earlier), you can follow the bug here: https://bugzilla.mozilla.org/show_bug.cgi?id=1577248. 
We can close this issue and I can add a new one once this is ready if you'd like, I think that's fine.

**Comment by chenba** [comment]
@juliastarkov Sounds good.  We'll get the changes so far merged, and once the illustration's ready, we can add it in another issue.
